.release/linux/package/etc/boundary.d/boundary.hcl

```diff
@@ -1,3 +1,112 @@
+# # Note that this is an example systemd file and is not intended to be functional as-is.
 # # Full configuration options can be found at https://www.boundaryproject.io/docs/configuration
 
+# # What follows is an example set up for a Boundary worker. Further down in the file you will
```

Comment:
Rather than put two configs in one file, a more common/usable approach is to have separate example configs in the same directory, that the user can copy, edit or symlink.  So here you could either:
* Keep the controller config here as the default in `boundary.hcl`, with a separate worker config in a different file in the `boundary.d` directory that could be swapped in or referred to in-place; or
* Have both configs in separate files in the `boundary.d` directory and make `boundary.hcl` a symlink to one of them.

Reply:
Does that make sense given that the user could theoretically want to run both the controller and the worker in the same process (using the same config file, as is done here)?

Reply:
I separated this into two files, with controller as boundary.hcl.